src/pages/junghyowon/Login/JungLogin.js

```diff
@@ -1,7 +1,103 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './JungLogin.scss';
 
 function JungLogin() {
-  return <div>정효원 로그인페이지</div>;
+  const [id, setId] = useState('');
+  const [password, setPw] = useState('');
+  const navigate = useNavigate();
+  const idPw = id.indexOf('@') !== -1 && password.length > '5';
+
+  const onCheckEnter = e => {
+    if (e.key === 'Enter') {
+      goToMain();
+    }
+  };
+
+  const goToMain = e => {
+    navigate('/JungMain');
+  };
+
+  const idValue = e => {
+    setId(e.target.value);
+  };
+
+  const pwValue = e => {
+    setPw(e.target.value);
+  };
```

Comment:
```suggestion
  const getUserInfo = e => {
    const { name, value} = e.target;
    setUserInfo({ ...userInfo, [name]:value });
  };
```
- 구조 분해 할당과 계산된 속성명을 활용하면 하나의 함수로도 할 수 있습니다.
- 그러기 위해선 Input에 name과 value 및 상태도 하나로 관리해주셔야 합니다.